pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -240,6 +240,9 @@ func (m *kubeGenericRuntimeManager) startContainer(ctx context.Context, podSandb
 	}
 
 	target, err := spec.getTargetID(podStatus)
+	if restartCount > 0 {
```

Comment:
nit: I'd suggest to move this and 2 following lines up, so this `if` statement is above `target, err := spec.getTargetID(podStatus)` line. This is more readable from my point of view.

Reply:
Alright, will do that and test the results.